### DISCUSSION TITLE
Different curve types per line in LineChart

### DISCUSSION THREAD

**Discussion by highviolet** [mantinedev/mantine#5679] [Feature requests]
The addition of charts to Mantine is great.
I'm currently migrating from using core recharts to use mantine instead.

One thing that I'm using in recharts is to plot multiple lines but with different curve types.
I.e. mixing monotone and step lines.

So it would be great to have another optional parameter to LineChartSeries that allows specifying the curve type per series.

**Comment by rtivital** [comment]
A feature to pass props per chart part is in the backlog.
